guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -451,8 +452,8 @@ Use this option if you have one of the following errors:
 . In the {Project} web UI, navigate to *Content* > *Products*.
 . Select the product containing the corrupted repository.
 . Select the name of a repository you want to synchronize.
-. From the *Select Action* menu, select *Advanced Sync*.
-. Select the option and click *Sync*.
+. To verify the checksum, click *Verify Content Checksum* from the *Action* menu.
```

Comment:
. To verify the checksum, click *Verify Content Checksum* from the *Select Action* menu.

Reply:
Orient the user first as per IBM style, so @mjivraja is slightly more correct here. Depending on the style guide, this rule changes, but we follow IBM here.